### PR TITLE
Multiple frames

### PR DESCRIPTION
**Pull request by joaompneves**
Made changes to support the capability to add inner views

### CODE REVIEW

File: ReactViewControl/ReactViewRender.cs [hunk 1]
```diff
@@ -1,46 +1,47 @@
...
+            Initialized,
+            PageLoaded,
+            Ready
+        }
+
+        internal const string ModulesObjectName = "__Modules__";
         private const string ComponentLoadedEventName = "ComponentLoaded";
 
-        private static readonly Assembly ResourcesAssembly = typeof(ReactViewResources.Resources).Assembly;
+        private static Assembly ResourcesAssembly { get; } = typeof(ReactViewResources.Resources).Assembly;
```
Comment (alvesmiguel1) [inline ReactViewControl/ReactViewRender.cs]: Do you really need this as a property (since the usages have not been changed)?

Comment (alvesmiguel1) [reply ReactViewControl/ReactViewRender.cs]: Same for properties below

Comment (joaompneves) [reply ReactViewControl/ReactViewRender.cs]: This is a readonly property... using the "new" c# syntax

Comment (alvesmiguel1) [reply ReactViewControl/ReactViewRender.cs]: 👍

**Pull request merged**